xml/System/String.xml

```diff
@@ -4105,7 +4105,13 @@ Examples of instantiating strings:
         <summary>Returns a value indicating whether a specified character occurs within this string.</summary>
         <returns>
           <see langword="true" /> if the <paramref name="value" /> parameter occurs within this string; otherwise, <see langword="false" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
```

Comment:
My understanding is that remarks don't show in intellisense. If we want this to show up in VS, we should move this change up into the summary.

Reply:
Just copying `Contains (string value)`. @BillWagner ?

Reply:
@BillWagner could you help with that question?